dev-lang/rust/rust-1.54.0.ebuild

```diff
@@ -303,6 +306,7 @@ src_configure() {
 		ninja = true
 		targets = "${LLVM_TARGETS// /;}"
 		experimental-targets = ""
+		use-libcxx = $(usex libcxx true false)
```

Comment:
there should be a dependency on libcxx, right?

Reply:
Done.